src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -2636,16 +2649,16 @@ namespace Akka.Dispatch.SysMsg
     }
     public sealed class Unwatch : Akka.Actor.INoSerializationVerificationNeeded, Akka.Dispatch.SysMsg.ISystemMessage
     {
-        public Unwatch(Akka.Actor.IActorRef watchee, Akka.Actor.IActorRef watcher) { }
-        public Akka.Actor.IActorRef Watchee { get; }
-        public Akka.Actor.IActorRef Watcher { get; }
+        public Unwatch(Akka.Actor.IInternalActorRef watchee, Akka.Actor.IInternalActorRef watcher) { }
```

Comment:
`Watch` and `Unwatch` can't work without `IInternalActorRef`, so I changed the signature of the message contents to reflect that. In general we've been very inconsistent about using `IInternalActorRef` where we should.